### DISCUSSION TITLE
Archery无法创建视图

### DISCUSSION THREAD

**Discussion by jesse1218** [hhyo/Archery#2802] [Q&A]
Archery版本：v1.10
使用Archery提交创建视图提示“不允许创建或使用视图 'test”
![screenshot-20240912-000933](https://github.com/user-attachments/assets/ae706fa4-eeca-4cf7-8149-fd69b5b03727)
查看sql上线限制也没有禁止创建视图
![screenshot-20240912-001012](https://github.com/user-attachments/assets/fde8b37b-5f81-4002-a126-f5f6d4c2463f)
感谢一切回答!

**Comment by Maxcj** [comment]
这个实际上是 goInception 限制的，可以修改 **config.toml** 这个文件
`enable_use_view=true`

具体可以参考以下链接
https://hanchuanchuan.github.io/goInception/zh/options.html#variables%E5%88%97%E8%A1%A8
